static/app/components/events/eventStatisticalDetector/breakpointChart.tsx

```diff
@@ -29,6 +29,52 @@ type EventBreakpointChartProps = {
   event: Event;
 };
 
+type BreakpointChartData = {
+  event: Event;
+  eventStatsReponse:
+    | {
+        data: EventsStatsData;
+        meta: MetaType;
+      }
+    | undefined;
+};
+function getBreakPointChartPropsFromData(chartData: BreakpointChartData) {
+  const event = chartData.event;
+  const {breakpoint} = event?.occurrence?.evidenceData ?? {};
+  const datetime = getDateTimeFromBreakPoint(breakpoint);
+
+  // The evidence data keys are returned to us in camelCase, but we need to
+  // convert them to snake_case to match the NormalizedTrendsTransaction type
+  const normalizedOccurrenceEvent = Object.keys(
+    event?.occurrence?.evidenceData ?? []
+  ).reduce((acc, key) => {
+    acc[camelToUnderscore(key)] = event?.occurrence?.evidenceData?.[key];
+    return acc;
+  }, {}) as NormalizedTrendsTransaction;
+
+  const p95Series = useMemo(
+    () =>
+      transformEventStats(
+        chartData.eventStatsReponse?.['p95(transaction.duration)']?.data ?? [],
+        generateTrendFunctionAsString(TrendFunctionField.P95, 'transaction.duration')
+      ),
+    [chartData.eventStatsReponse]
+  );
+
+  return {
+    datetime,
+    normalizedOccurrenceEvent,
+    p95Series,
+  };
+}
+
+function getDateTimeFromBreakPoint(breakpoint: number) {
+  return useRelativeDateTime({
+    anchor: breakpoint,
+    relativeDays: RELATIVE_DAYS_WINDOW,
+  });
+}
```

Comment:
`useRelativeDateTime` uses react hooks. If you wrap it up within a regular function, it can easily be used in wrong ways.